### PR TITLE
Scaling up the recipe with damip request

### PR DESCRIPTION
**Pull request by jbusecke**
This PR implements the first query from #3. I used my WIP parsing logic locally to generate the list of dataset_ids in the recipe and ran some quick QC locally: 
- There are two datasets (commented out) for which the ESGF API does not provide any urls. We need a good way to handle these cases (since they might be possible just intermittent). Ideas welcome.

- There is an issue with some of the datasets being provided in fairly small files. @cisaacstern and I have talked about this and this can probably be resolved by expanding the dynamic kwargs logic to not just subset files but also combine them with `inputs_per_chunk` being determined dynamically aswell.

But for now lets try these and see if that helps @rebeccaherman1

### CODE REVIEW

**Comment by jbusecke** [comment]
Ok I think I have figured the netcdf detection out! Thanks to @rabernat @cisaacstern and [this doc](https://docs.unidata.ucar.edu/netcdf-c/current/faq.html#How-can-I-tell-which-format-a-netCDF-file-uses). 

I have reduced the number of datasets to one member per model (assuming that the file format for the output does not change between modelling centers...

File: feedstock/recipe.py [hunk 1]
```diff
@@ -330,6 +330,12 @@ def choose_chunksize(
     else:
         return output_chunksizes[0]
 
+def is_netcdf3(url:str) -> bool:
```
Comment (cisaacstern) [inline feedstock/recipe.py:333]: nice!

**Comment by cisaacstern** [comment]
> Should I uncomment the remaining instance ids?

Just to keep this intelligible, could we merge this, and then open a new PR which uncomments the remaining IDs?

**Comment by jbusecke** [comment]
Yayyyy the netcdf3 detection works! Ill merge this now and try to push further on this throughout the day.

**Pull request merged**